website/docs/r/subnet.html.markdown

```diff
@@ -112,6 +123,7 @@ Class - fvSubnet
 - `annotation` - (Optional) Annotation for object subnet.
 - `description` - (Optional) Description for object subnet.
 - `ctrl` - (Optional) The list of subnet control state. The control can be specific protocols applied to the subnet such as IGMP Snooping. Allowed values are "unspecified", "querier", "nd" and "no-default-gateway". Default is "nd". NOTE: "unspecified" should't be used along with other values.
+- `ip_data_plane_learning` - (Optional) Knob to disable IP Dataplane Learning for Host(/32, /128) and for BD Subnet. Type - String.
```

Comment:
```suggestion
- `ip_data_plane_learning` - (Optional) Flag to enable/disable ip-data-plane learning for the Subnet object. Allowed values are "enabled" and "disabled" and default value is "enabled". Type: String.
```

Reply:
Could you adjust the other attributes in argument spec to align with this format:

1. change "object subnet" to "the Subnet object"
2. append Type: String. at the end of the sentences
3. start sentences with "The"
4. provide allowed values and default value as 1 sentence.

Reply:
In regards to suggestion - I got attribute description from MIM model docs as I've noticed this is how attributes are described in most of the cases here. Of course I've adjusted it to suggested description above.

Please tell me if there's a specific rule for writing a description for it - previously I thought the MIM model is like a source of truth.

Thanks

Reply:
I see fvSubnet docs should be updated to meet the new standard, I can update this also.